apps/prairielearn/src/tests/issues.test.ts

```diff
@@ -45,36 +45,43 @@ function doTest(issuesUrl: string, label: string) {
         .attr('value');
       assert(typeof variantId === 'string');
 
-      const form = {
-        __action: 'report_issue',
-        __csrf_token: csrfToken,
-        description: 'Something bad happened',
-        __variant_id: variantId,
-      };
       res = await fetch(questionUrl, {
         method: 'POST',
-        body: new URLSearchParams(form),
+        body: new URLSearchParams({
+          __action: 'report_issue',
+          __csrf_token: csrfToken,
+          __variant_id: variantId,
+          description: 'Something bad happened',
+        }),
       });
       $ = cheerio.load(await res.text());
 
       const result = await sqldb.queryAsync(sql.select_open_issues, []);
       assert.equal(result.rowCount, 1, 'Expected one open issue');
     });
 
-    step('should close question', async () => {
+    step('should close all matching issues', async () => {
```

Comment:
This test still checks for all issues. I think it would be a good idea to include a separate test for a partial match, when there is a query in place. I won't block on it though.

Reply:
Done in e4f183c. Can you give it a look and make sure it's testing what you wanted?